MichiganDaily/bore
# Add `BarChart` example

#### What's this PR do?

Adds an example of how to use `bore.BarChart`.

#### Are there any relevant screenshots?

N/A

#### Why are we doing this? How does it help us?

Helps people use `bore.BarChart`.

#### How should this be manually tested?

Try out the example. See the README for [installing `bore`](https://github.com/MichiganDaily/bore#using-bore).

#### Are there any smells or added technical debt to note?

No

#### What are relevant issues or links?

N/A

#### Have you done the following, if applicable:
